views/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Index</title>
+</head>
+<body>
+<form action='http://localhost:3000/getAll'><input type="submit" value="Get All"></form>
+<form action='http://localhost:3000/insertForm.html'><input type="submit" value="Insert"></form>
+<form action='http://localhost:3000/updateForm.html'><input type="submit" value="Update"></form>
```

Comment:
these could all be anchor tags for simplicity, a button implies some sort of data exchange with the server